language/move-prover/bytecode/src/stackless_bytecode_generator.rs

```diff
@@ -1345,7 +1345,13 @@ impl<'a> StacklessBytecodeGenerator<'a> {
                         .collect::<Vec<BigUint>>();
                     Constant::AddressArray(b)
                 }
-                _ => unimplemented!("Not yet supported constant vector type: {:?}", ty),
+                _ => {
```

Comment:
Do we want to keep `Type::Primitive(PrimitiveType::Address)` arm above or fold it into the more generic handling of constant vectors?

Reply:
I would like to keep it for now in case my PR may break existing working code :)